.github/workflows/wcag_test.yml

```diff
@@ -5,20 +5,7 @@ on:
     branches: [develop, hotfix/*, release/*, support/*]
 
 jobs:
-  WCAG-test:
-    runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [14.x]
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v2
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm run setup-test-site
-    - run: npm run build-test-site
-    - run: npm run wcag
\ No newline at end of file
+  call_wcag_test:
+    uses: yext/slapshot-reusable-workflows/.github/workflows/wcag_test.yml@v1
+    with:
+      build_script: 'npm run setup-test-site && npm run build-test-site'
```

Comment:
nit: some buid_scripts like percy_snapshots and run_tests omit the quotes, can this be remove to be consistent?